scheduler/scheduler-api/src/main/resources/org/ow2/proactive/scheduler/common/xml/schemas/jobdescriptor/dev/schedulerjob.xsd

```diff
@@ -449,6 +475,16 @@
       </xs:annotation>
     </xs:attribute>
   </xs:attributeGroup>
+  <xs:attributeGroup name="taskRetryDelay_t">
+    <xs:attribute name="taskRetryDelay" use="required">
+      <xs:annotation>
+        <xs:documentation>Specifies how long to wait before restart the task if an error occurred.</xs:documentation>
```

Comment:
`This parameter specifies how long to wait before restarting the task if an error occurred.`

Reply:
Done.